tests/rustdoc-gui/docblock-code-block-line-number.goml

```diff
@@ -68,20 +75,18 @@ call-function: ("check-colors", {
 // Now, try changing the setting dynamically. We'll turn it off, using the settings menu,
 // and make sure it goes away.
 
-// First, open the settings menu.
-click: "rustdoc-toolbar .settings-menu"
-wait-for: "#settings"
-assert-css: ("#settings", {"display": "block"})
+call-function: ("switch-line-numbers-setting", {"expected_status": "false"})
+assert: ".digits-1.hide-lines"
 
-// Then, click the toggle button.
-click: "input#line-numbers"
-wait-for: ".digits-1.hide-lines"
-assert-local-storage: {"rustdoc-line-numbers": "false" }
+// The line numbers not being displayed, their "space" should have disappear as well.
+assert-position: (".example-wrap code .macro", {"x": |span_x_pos|})
 
 // Finally, turn it on again.
-click: "input#line-numbers"
+call-function: ("switch-line-numbers-setting", {"expected_status": "true"})
 wait-for: ".digits-1:not(.hide-lines)"
-assert-local-storage: {"rustdoc-line-numbers": "true" }
+
+// The line numbers are being displayed, their "space" should be back.
+assert-position-false: (".example-wrap code .macro", {"x": |span_x_pos|})
 
 // Same check with scraped examples line numbers.
```

Comment:
this comment seems somewhat nonsensical now.

Reply:
it seems like a similar check, but not exactly the same, in particular the padding checks aren't done for the first case.

Reply:
They're checked in the `check-colors` function (which is badly named ^^').